app/assets/javascripts/modules/MultiTableFilter.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * # Multiple table filter
+ *
+ * Filters across one or more tables, and can also hide the elements associated with a table.
+ *
+ * Requires an element to have a data-dough-component="MultiTableFilter" attribute and an ID.
+ *
+ * Makes use of jquery.fastLiveFilter.
+ * See test fixture for sample markup - /spec/js/fixtures/MultiTableFilter.html
+ */
+
+define(['jquery', 'jqueryFastLiveFilter', 'DoughBaseComponent'],
+       function($, jqueryFastLiveFilter, DoughBaseComponent) {
+  'use strict';
+
+  var MultiTableFilterProto,
+      defaultConfig = {
+        filterTargetSelector: '.js-filterable',
+        filterFieldSelector: '.js-filter-input',
+        filterListSelector: '.js-filter-rows',
+        filterWrapperSelector: '.js-filter-group'
+      };
+
+  function MultiTableFilter($el, config) {
+    MultiTableFilter.baseConstructor.call(this, $el, config, defaultConfig);
+  }
+
+  /**
+   * Inherit from base module, for shared methods and interface
+   * @type {[type]}
+   * @private
+   */
+  DoughBaseComponent.extend(MultiTableFilter);
```

Comment:
This is a great example of how we could do better at updating everyone with changes, but components now need to be named else you'll get warnings in the console with the latest version of Dough. Its simple to do: https://github.com/moneyadviceservice/dough/commit/8479406c3a49f5ca8af43a408c2c07b9b3f0c469